core/storage/user/gae_models.py

```diff
@@ -313,15 +323,15 @@ def export_data(user_id):
             user_id: str. The user_id denotes which user's data to extract.
 
         Returns:
-            dict or None. A dict with two keys, 'completed_exploration_ids'
+            dict. A dict with two keys, 'completed_exploration_ids'
             and 'completed_collection_ids'. The corresponding values are
             lists of the IDs of the explorations and collections,
             respectively, which the given user has completed. If the
-            user_id is invalid, returns None.
+            user_id is invalid, returns an empty dict.
```

Comment:
I'm confused about this. If the user ID is invalid then shouldn't None be returned or an exception be thrown? This is as opposed to if the user ID is valid but the data doesn't exist here because they haven't done anything, but in that case I wouldn't call it "invalid"...

(Possibly ditto elsewhere. We need to be careful about ensuring that the docstrings are actually correct in their descriptions.)

Reply:
Changed the docstring.